4.0.0/migration.md

```diff
@@ -27,135 +146,148 @@ The migration process is initiated by the Installer if the following requirement
+
+9. Ensure that MySQL deployments (non-HA setup) or stateful sets (HA setup) and services are deleted:
 
         deploy.go:1664] removing mysql deployments, stateful sets and services
 
-   At this point, the migration is successfully concluded and installation will continue as before.
-   For the following runs, installer will always generate the manifest for `mysql-to-postgresql-migration` and `mysql-latest-migrations jobs:`.
-   Even if you keep generating the manifests, the migration will not run again if it is once completed.
+   At this point, the migration is successfully concluded and installation will continue as usual.
+
 
 10.  As the PVC is retained by default, delete it after the migration is completed.
 
+
+**For the following runs, Installer will always generate the manifests for `mysql-to-postgresql-migration` and `mysql-latest-migrations` jobs: even if so, the migration will not run again if it has completed once.**
+
+
 ## Troubleshooting
 
-### PostgreGRE Initialization Failed
+### PostgreSQL Initialization Job Failed
+
+The whole migration process will continue even if after 10 checks the `postgres-init-job` job has not finished correctly:
 
-Running PostgreGRE init job even after 10 checks indicates that the `postgres-init-job` job has not finished correctly.
+        deploy.go:1711] removing postgres db init job
+        deploy.go:1714] running postgres db init job
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 1/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 2/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 3/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 4/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 5/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 6/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 7/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 8/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 9/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 10/10)
+        deploy.go:1726] waiting postgres db init job to finish... (attempt 10/10)
+        deploy.go:1730] postgres-init-job failed
 
-     deploy.go:1711] removing postgres db init job
-     deploy.go:1714] running postgres db init job
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 1/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 2/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 3/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 4/10)
-     deploy.go:1726] waiting postgres db init job to finish... (attempt 5/10)
 
+1. You can stop the installer with `CTRL + c` because the migration will fail later in any case.
```

Comment:
Stop the installer with `CTRL + c` because the migration will fail later in any case.